components/proxy/src/main/java/com/hotels/styx/StyxServer.java

```diff
@@ -87,14 +89,27 @@ private static StyxServer createStyxServer(String[] args) {
             LOG.info("Styx configFileLocation={}", startupConfig.configFileLocation());
             LOG.info("Styx logConfigLocation={}", startupConfig.logConfigLocation());
 
-            Configuration configFromFile = new ConfigurationParser.Builder<YamlConfiguration>()
+            YamlConfiguration yamlConfiguration = new ConfigurationParser.Builder<YamlConfiguration>()
                     .format(YAML)
                     .overrides(System.getProperties())
                     .build()
                     .parse(configSource(startupConfig.configFileLocation()));
 
+            if (skipServerConfigValidation()) {
```

Comment:
The validation should be called from inside the `ConfigurationParser` rather than adding more coupling between `StyxServer` and YAML, and adding bloat back into a class we only just cleaned up.